internal/oc/release.go

```diff
@@ -148,17 +151,33 @@ func (r *release) GetMajorMinorVersion(log logrus.FieldLogger, releaseImage stri
 
 func (r *release) GetReleaseArchitecture(log logrus.FieldLogger, releaseImage string, releaseImageMirror string, pullSecret string) (string, error) {
 	var cmd string
+	var cmdMultiarch string
 	if releaseImage == "" && releaseImageMirror == "" {
 		return "", errors.New("no releaseImage nor releaseImageMirror provided")
 	}
 	if releaseImageMirror != "" {
 		cmd = fmt.Sprintf(templateImageInfo, releaseImageMirror)
+		cmdMultiarch = fmt.Sprintf(templateSkopeoDetectMultiarch, releaseImageMirror)
 	} else {
 		cmd = fmt.Sprintf(templateImageInfo, releaseImage)
+		cmdMultiarch = fmt.Sprintf(templateSkopeoDetectMultiarch, releaseImage)
 	}
-	imageInfoStr, err := execute(log, r.executer, pullSecret, cmd)
+
+	imageInfoStr, err := execute(log, r.executer, pullSecret, cmd, ocAuthArgument)
 	if err != nil {
-		return "", err
+		// TODO(WRKLDS-222) At this moment we don't have a better way to detect if the release image is a multiarch
+		//                  image. Introducing skopeo as an additional dependency, to be able to manually parse
+		//                  the manifest. https://bugzilla.redhat.com/show_bug.cgi?id=2111537 tracks the missing
+		//                  feature in oc cli.
+		skopeoImageRaw, err2 := execute(log, r.executer, pullSecret, cmdMultiarch, skopeoAuthArgument)
+		if err2 != nil {
+			return "", errors.Errorf("failed to parse image, oc: %v, skopeo: %v", err, err2)
```

Comment:
Maybe 'failed to inspect image' is a clearer error in this case.

Reply:
Sure, makes sense